### DISCUSSION TITLE
Underscore brings me to the start of a line

### DISCUSSION THREAD

**Discussion by uduse** [VSCodeVim/Vim#7550] [Q&A] [answered]
Somehow when I press `_` (aka `shift + -`) in command mode, it brings me to the start of the line instead of using the black hole register. I couldn't find any related vim setting or key binding that may cause this problem. Anyone has an idea what's happening here?

**Comment by J-Fields** [comment, marked as answer]
This is expected behavior: [`:help _`](https://vimhelp.org/motion.txt.html#_). Maybe you're used to having a remap like `nnoremap _ "_`?

**Reply by uduse** [reply]
Thank you for the reply! I was reading the docs and I thought `"` is something else.
